spec/utils/data_seeder_spec.rb

```diff
@@ -1,16 +1,20 @@
 require 'rails_helper'
 
-RSpec.describe 'db:seed' do
+RSpec.describe DataSeeder do
   it 'creates DumpType and DumpFileType entries' do
     DumpType.all.map(&:destroy)
     DumpFileType.all.map(&:destroy)
-    Rails.application.load_seed
+    seeder = described_class.new
+    seeder.generate_dump_types
+    seeder.generate_dump_file_types
     expect(DumpType.count).not_to be_zero
     expect(DumpFileType.count).not_to be_zero
   end
 
   it 'is idempotent' do
-    Rails.application.load_seed
+    seeder = described_class.new
+    seeder.generate_dump_types
+    seeder.generate_dump_file_types
     expect { Rails.application.load_seed }.not_to change { DumpType.count }
```

Comment:
Could we try this with `expect {seeder.generate_dump_types}.not_to`? I think it would be more specific to the class now under test.
Also for the line below.

Reply:
Thanks. Missed that. Tired at the end of the day.